types/pg/index.d.ts

```diff
@@ -153,7 +153,7 @@ export class Pool extends events.EventEmitter {
     end(callback: () => void): void;
 
     query<T extends Submittable>(queryStream: T): T;
-    query(queryConfig: QueryArrayConfig): Promise<QueryArrayResult>;
+    query(queryConfig: QueryArrayConfig, values?: any[]): Promise<QueryArrayResult>;
```

Comment:
Do you want to use the new TypeScript arg features for this? `any` seems dangerous here.

Reply:
Not sure I understand the suggestion. I just followed (copied) the existing definition of `query(queryTextOrConfig: string | QueryConfig, values?: any[]): Promise<QueryResult>`

BTW, according to code, 'values' can either be a _function_ or an _array_ of values of _any_ type. We can define values param as: function | any[], not sure if this will be correct.